logprep/processor/domain_label_extractor/rule.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, filter_rule: FilterExpression, domain_label_extractor_cfg: di
         self._target_field = domain_label_extractor_cfg["target_field"]
         self._output_field = domain_label_extractor_cfg["output_field"]
 
-    def __eq__(self, other: 'DomainLabelExtractorRule') -> bool:
+    def __eq__(self, other: "DomainLabelExtractorRule") -> bool:
         return (other.filter == self._filter) and (self._compare_set == other.compare_set)
```

Comment:
my editor says, that there is no `_compare_set` attribute on self. And as I see, there is no test for `__eq__` method in `rule.py`

Reply:
fixed.